### PR TITLE
Added floor

### PR DESCRIPTION
**Pull request by SolMendiola**
<!---

# :star: Feature

---

#### :pencil2: Description:

Added floor and changed to stream 

---

### CODE REVIEW

File: lib/core/repository/tip_repository.dart [hunk 1]
```diff
@@ -1,11 +1,39 @@
...
 
 class TipRepository {
   final TipRemoteSource _tipRemoteSource;
 
-  TipRepository(this._tipRemoteSource);
+  final TipsLocalSource _tipsLocalSource;
+
+  //TipRepository(this._tipRemoteSource, this._tipsLocalSource);
 
   //TODO: change to a stream in next PR with floor
```
Comment (mirland) [inline lib/core/repository/tip_repository.dart:18]: rm

**Review by f7deleon — COMMENTED** [review]
now that we have our own tips it would be good to delete all related to projects

**Review by f7deleon — APPROVED** [review]

File: lib/core/source/database.dart [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+import 'package:flutter_template/core/source/localSource/tips_local_source.dart';
+part 'database.g.dart';
+
+@Database(version: 1, entities: [TipDbEntity])
+abstract class AppDatabase extends FloorDatabase {
+  TipsLocalSource get tipsLocalSource;
+
+  Future<void> clearAllTables() async {
+    await database.delete('tips');
+  }
```
Comment (mirland) [inline lib/core/source/database.dart:14]: use lamda

File: lib/ui/tips/tips_cubit.dart [hunk 3]
```diff
@@ -14,10 +16,20 @@ part 'tips_state.dart';
...
   final GeneralErrorHandler _errorHandler;
+  late StreamSubscription<List<Tip>> tips;
   final TipRepository _tipRepository = DiProvider.get();
 
-  TipCubit(this._errorHandler) : super(const TipsBaseState.state()) {
-    unawaited(getTips());
+  TipCubit(this._errorHandler)
+      : super(
+          const TipsBaseState.state(),
+        ) {
```
Comment (mirland) [inline lib/ui/tips/tips_cubit.dart:25]: remove comma

**Pull request merged**